[GLib] Tools/glib/apply-build-revision-to-files.py fails on Git worktrees

aperezdc opened the pull request:
#### 56353fd580f3cffa5d2abe69b552016f933f14b6
<pre>
[GLib] Tools/glib/apply-build-revision-to-files.py fails on Git worktrees
<a href="https://bugs.webkit.org/show_bug.cgi?id=242176">https://bugs.webkit.org/show_bug.cgi?id=242176</a>

Reviewed by Philippe Normand.

* Tools/glib/apply-build-revision-to-files.py:
(get_build_revision): Cover the case where .git is a regular file, as
used for worktrees.

Canonical link: <a href="https://commits.webkit.org/251991@main">https://commits.webkit.org/251991@main</a>
</pre>

philn reviewed (approved):
Would be nice to get rid of the SVN stuff in that script at some point :)

aperezdc committed:
[GLib] Tools/glib/apply-build-revision-to-files.py fails on Git worktrees
https://bugs.webkit.org/show_bug.cgi?id=242176

Reviewed by Philippe Normand.

* Tools/glib/apply-build-revision-to-files.py:
(get_build_revision): Cover the case where .git is a regular file, as
used for worktrees.

Canonical link: https://commits.webkit.org/251991@main
